### ISSUE TITLE
When I am trying to create any factory its taking very huge time

### ISSUE THREAD

**Issue by chandanchoubey112** [thoughtbot/factory_bot_rails#398]
### Description
I am not sure but simplest Factory is also taking huge time. And this is only one spec. 

```
t = Benchmark.measure { create(:user, email: 'test11@gmail.com') }
t.total # 75.06367000000319
```

Using mac.
**MacBook Pro (Retina, 15-inch, Late 2013)**
**Processor: 2.3 GHz Quad-Core Intel Core i7**
**memory: 16 GB 1600 MHz DDR3**

### System configuration
**factory_bot_rails version**:  factory_bot_rails (6.2.0)
**factory_bot version**:  factory_bot (~> 6.2.0)
**rails version**:  6.1
**ruby version**: 2.7.2

**Comment by composerinteralia** [comment]
Without seeing your specific factory and model code, I'm not sure how to be of much help. You could maybe narrow it down a bit by benchmarking `build(:user, email: 'test11@gmail.com')` to see if the slowdown is happening when building the records or when saving them.

**Comment by chandanchoubey112** [comment]
@composerinteralia  But I need a a record which will have `id` and all, Build will initialise a object wont create.
Build wont solve my problem

**Comment by composerinteralia** [comment]
Makes sense that you need create instead of build for your tests to pass. I was recommending you change to build temporarily so you can get a benchmark for it. If build is fast but create is slow, then you know the slow part is happening during save. That might be helpful in narrowing down on what the problem is.

To help more than that, we’d need either a reproduction script or a sample application that can reproduce this performance problem.

thank you!
